docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -3243,68 +3243,68 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Reason:**
 
-  TODO
+  (_operatingSystemName_) is not a supported platform for _componentName_. Skipping _pluginid_... Supported: _requiredOperatingSystem_
 
   **Action:**
 
-  TODO
+  Contact the author of the component, or a system administrator.
 
 
 
 ### ZWED0154E":"(%s) is not a supported architecture for %s. Skipping (%s)... Supported: %s
 
   **Reason:**
 
-  TODO
+  (_operatingSystemName_) is not a supported platform for _componentName_. Skipping _pluginid_... Supported: _requiredOperatingSystem_
 
   **Action:**
 
-  TODO
+  Contact the author of the component, or a system administrator.
 
 
 
 ### ZWED0155E":"(%s) is not a supported endpoint for %s. Skipping (%s)... Supported: %s
 
   **Reason:**
 
-  TODO
+  (_url_) is not a supported endpoint for _componentName_. Skipping _pluginid_... Supported: _urls_
```

Comment:
you're just putting the message description in the reason.
the message description of all the lines in this PR are still broken.

you have
```
### ZWED0155E":"(%s) is not a supported endpoint for %s. Skipping (%s)... Supported: %s
```

you should have

```
### ZWED0155E

    (_url_) is not a supported endpoint for _componentName_. Skipping _pluginid_... Supported: _urls_

   **Reason:**

    something insightful
```

and so on, for all the messages.